GoogleIMA/GoogleIMA.json

```diff
@@ -1,3 +1,3 @@
 {
-	"3.9.0": "https://github.com/dowjones-mobile/external-binaries/raw/master/GoogleIMA/ima-ios-v3.9.0.zip"
+	"3.11.4": "https://github.com/dowjones-mobile/external-binaries/raw/master/GoogleIMA/ima-ios-v3.11.4.zip"
```

Comment:
This must be an addition not replacement.
You should never delete the old version from anywhere.
What if someone wants to build "older" version of the app?

Reply:
@ChadDowJ also please try to use more meaningful messages of your commit. `test` is not very descriptive.